helm/README.md

```diff
@@ -0,0 +1,123 @@
+## Gluu Server
+
+[Gluu server](https://www.google.com) is an open-source IAM server that sync backend identities, leverage external IDPs, and achieve SSO, 2FA and more.
+
+## TL;DR;
+
+`helm install beta/gluu`
+
+## Introduction
+
+This chart bootstraps a Gluu Server deployment on a kubernetes cluster using Helm package manager.  
+It also packages other components/services that make up Gluu Server.
+
+## Prerequisites
+
+- kubernetes 1.5+ 
```

Comment:
K8 1.5+ only ? This will be quite the issue. AWS EKS supports now up to 1.13 only and we should be able to apply helm to V 1.13 and above.